osu.Server.Spectator/ChatFilters.cs

```diff
@@ -21,18 +28,57 @@ public ChatFilters(IDatabaseFactory factory)
 
         public async Task<string> FilterAsync(string input)
         {
-            if (filters == null)
+            if (!filtersInitialised)
+                await initialiseFilters();
+
+            if (blockRegex?.Match(input).Success == true)
+                throw new InvalidStateException(string.Empty);
```

Comment:
I didn't see much benefit to providing an actual exception message here.

This is how this looks client-side:

![osu_2024-05-01_09-26-15](https://github.com/ppy/osu-server-spectator/assets/20418176/49fff0f8-7f24-46ff-a15a-da6d6c804aab)

Reply:
How does it show if there is a message? Just as part of the exception description?

Reply:
Yeah it just gets appended at the end.

![1714549968](https://github.com/ppy/osu-server-spectator/assets/20418176/56539bb6-6ff2-4275-aa21-91cee1cb2131)

Reply:
Maybe better than nothing.. You just know users are going to report as a bug otherwise 😅

Reply:
Yeah sure. Not sure if you want me to apply something or are going to do so yourself, given the review self-request below.

Reply:
I'll grab it